docker/docker-compose.lisk.core.yml

```diff
@@ -0,0 +1,79 @@
+version: '3'
+services:
+  lisk:
+    image: lisk/core:3.0.0-beta.4
+    ports:
+      - 4000
+      - 5001
+    networks:
+      - lisk
+    restart: always
+    command: ['--network=devnet', '--api-ws', '--api-ws-port=5001', '--enable-forger-plugin', '--enable-http-api-plugin', '--http-api-plugin-port=4000', '--http-api-plugin-whitelist=172.16.0.0/12']
+    environment:
+      - LISK_API_PUBLIC=true
```

Comment:
This part seems to be enough. It can be easily combined with others.

Reply:
Done!